src/index.js

```diff
@@ -0,0 +1,26 @@
+const repl = require("repl");
+
+function getRandomNumber (max) {
+   return Math.floor(1 + Math.random() * (+max + 1 - 1));
```

Comment:
what is the point of adding 1 and then subtracting it?

Reply:
it's a mistake, already fixed.